change/js/app.js

```diff
@@ -2,6 +2,49 @@ console.log("Sanity Check: JS is working!");
 
 $(document).ready(function() {
 
-  $("#time").text(Date.now());
+    var initialize = function() {
+        $("#time").text(Date.now());
+        $("#left").val(0);
+        $("#right").val(0);
+        $("#total").val(0);
+    };
+    initialize();
+
+    var totalValue = function() {
+        return parseInt($("#total").val());
+    };
+    var leftValue = function() {
+        return parseInt($("#left").val());
+    };
+    var rightValue = function() {
+        return parseInt($("#right").val());
+    };
+    console.log(typeof(totalValue()));
+
```

Comment:
Good stuff, but please define your functions using this format/syntax:

function totalValue(){
  return parseInt($("#total").val());
}